Adds 2 new bundles for Curator Beacon - Sword Saint and Carp Hunter

Hardly3D opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This PR adds two new bundles for the Curator's heroic beacon.

Ported from
https://github.com/skyrat-SS13/Skyrat-tg/pull/7187
https://github.com/tgstation/tgstation/pull/50020 (sans the taming stuff)

"Carp Hunter, Wildlife Expert - 2506"
Comes with an old carp spacesuit which is an inferior version of an actual one, a carp mask, a new hunting knife and a bag containing raw meat.

![OQpNO3DxRf](https://user-images.githubusercontent.com/66234359/147300088-42d9576a-21c9-491d-aca9-483b6485323d.png)

"Sword Saint, Wandering Vagabond - 1600's."
A weeb bundle that comes with a downgraded Katana (pretty much an reskinned claymore), a new kamishimo, rice hat and sandals.

![Ey6z5EiIL6](https://user-images.githubusercontent.com/66234359/147300223-6c46f1e9-912d-437a-a411-f81b865cdca0.png)


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
More variety is good for the game.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:Hardly
add: Adds Carp Hunter and Sword Saint into the Curator's Heroic Beacon
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Hardly3D committed:
Choice Beacon now uses TGUI

Ported from https://github.com/tgstation/tgstation/pull/61940

Hardly3D committed:
Adds The Carp Hunter Bundle

Ported from https://github.com/tgstation/tgstation/pull/50020 sans the taming features

Hardly3D committed:
Adds Ronin Bundle

Ported from https://github.com/Skyrat-SS13/Skyrat-tg/pull/7187

Also fixed some errors

ivanmixo commented on code/game/objects/items/kitchen.dm:
```diff
@@ -89,8 +89,12 @@
+	set_butchering()
+
+	///Adds the butchering component, used to override stats for special cases
+/obj/item/kitchen/knife/proc/set_butchering()
```
This proc is unnecesarry.

Can just add the component in the `ComponentInitialize` instead of making an unnecesarry proc which is just one line.

Hardly3D replied:
Yup, I have reverted that, should be less spaghetti now.

zeskorion commented on code/game/objects/items/religion.dm:
```diff
@@ -389,3 +389,10 @@
+/obj/item/katana/weak/curator
+	desc = "An ancient Katana. Forged by... Well, it doesn't really say, but surely it's authentic! And sharp to boot!"
+	force = 15
+	block_power = 30
```
this should be 0, not even chap gets a katana with this block power. Katanas have weaker blocking, because they are able to block projectiles

zeskorion commented on code/game/objects/items/storage/boxes.dm:
```diff
@@ -960,6 +960,15 @@
+
+/obj/item/storage/box/papersack/meat/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/reagent_containers/food/snacks/meat/slab(src)
```
why isnt this carp meat?

BrumalLake replied:
it's meant to be for taming and carps aren't cannibals but we don't have it at all so the bag doesn't deserve a place in the kit right now

Hardly3D replied:
Yep, it has meat with taming in mind, I should as well remove it from the bundle since bee doesn't have it.

zeskorion commented on code/modules/clothing/spacesuits/miscellaneous.dm:
```diff
@@ -396,6 +396,11 @@ Contains:
 	if (user.head == src)
 		user.faction -= "carp"
 
+/obj/item/clothing/suit/space/hardsuit/carp/old
```
this is objectively better than the curator's spacesuit kit. Perhaps give that kit a void jetpack to compensate?

zeskorion commented on code/game/objects/items/kitchen.dm:
```diff
@@ -129,6 +133,15 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_price = 60
 
+/obj/item/kitchen/knife/hunting
```
this has a force of 10 at the moment. it looks cool and deadly, maybe give it a force of 12 instead of 10? a bit better than a kitchen knife, not as good as a combat knife

zeskorion commented on code/modules/clothing/under/costume.dm:
```diff
@@ -145,6 +145,14 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 
+/obj/item/clothing/under/costume/kamishimo
```
most of the curator costumes have some level of armor.

PowerfulBacon replied:
no they dont

zeskorion replied:
Look at the jacket for the tomb raider costume.  The jumpsuits arent armores, being undersuits

zeskorion replied:
wait, im stupid, this is an undersuit... for some reason

PowerfulBacon commented on code/modules/clothing/spacesuits/miscellaneous.dm:
```diff
@@ -396,6 +396,11 @@ Contains:
 	if (user.head == src)
 		user.faction -= "carp"
 
+/obj/item/clothing/suit/space/hardsuit/carp/old
```
This is basically the traitor hardsuit considering slowdown doesn't matter in space. I'd nerf the armour values since space protection is already pretty strong

Hardly3D replied:
Oh, that wasn't intentional, I'll go give it a nerf.

Hardly3D committed:
Deleted meat papersack

Original PR had meat papersack with taming in mind but since we don't have that here I may as well just get rid of it

Hardly3D committed:
Balancing

Removed armor from carp suit
Nerfed katana's blocking and removed armor penetration
Slightly buffed hunting knife
Added katana into anime phobia's item list

Hardly3D committed:
Slightly buffed hunting knife with less spaghetti

Hardly3D committed:
I hate merge conflicts

Hardly3D committed:
Merge remote-tracking branch 'upstream/master' into curatorfun

Archanial commented on code/modules/clothing/spacesuits/miscellaneous.dm:
```diff
@@ -396,6 +396,13 @@ Contains:
+	name = "battered carp space suit"
+	desc = "It's covered in bite marks and scratches, yet seems to be still perfectly functional."
+	slowdown = 1
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0, "stamina" = 0)
```
Give it normal armor values instead of 0

Hardly3D replied:
By that I assume you mean buffing the suit's armor or give it the same armor as the original carp suit?

Archanial replied:
No, I mean normal as in like the other space suit curator has

Archanial reviewed (changes requested):
Fix conflicts

Hardly3D committed:
Merge remote-tracking branch 'upstream/master' into curatorfun

Hardly3D committed:
Slightly buffed old carp suit's armor, remove item_color from kamishimo.

Hardly3D committed:
Give carp suit same armor as NASA voidsuit

Hardly3D committed:
Add helmet for old carp suit.
